src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-undef */
+// Actions
+const LOAD = 'bookstore/widgets/LOAD';
+const CREATE = 'bookstore/widgets/CREATE';
+const UPDATE = 'bookstore/widgets/UPDATE';
+const REMOVE = 'bookstore/widgets/REMOVE';
```

Comment:
- [ ] Kindly make sure that each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), meaning that all action types for each reducer have to be unique. This can be implemented by changing the action type to something like the following:
```javascript
// Instead of having the following action type 
const LOAD = 'bookstore/widgets/LOAD';

// It can be changed to 
const LOAD = 'bookstore/books/LOAD';
```